### ISSUE TITLE
Pokemon X and Y - Lag at Pokemon Amie and always Crash

### ISSUE THREAD

**Issue by lucariokalos** [Lime3DS/Lime-3DS-Emulator#79]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

2108

### Description of Issue

When I play game its very smooth but when I enter in Pokemon Amie, it lags a lot and always crash.
My phone specs:

Name: Samsung Galaxy s22 Ultra
CPU: Snapdragon gen 1
Ram: 12gb
Storage: 256 gb
Android version 14

### Expected Behavior

Go to Pokemon Amie on X and Y and see

### Reproduction Steps

1. Enter Pokemon Amie
2. Wait the camera activate
3. When canera activate it will lag a lot
4. Eventually it will crash unexpectedly

### Log File

Nothing

### System Configuration

CPU: snapdragon gen 1
GPU/Driver:
RAM: 12 gb
OS: Android 14

**Comment by OpenSauce04** [comment]
This is possibly related to a seperate camera issue that I recently discovered where any instance of the camera feed being shown will result in rapid flickering

**Comment by Raffadanttas** [comment]
My pokemon x always close when im playing, i dont know what to do, for an hour the game runs normal then out of nowhere it closes

**Comment by Haisom** [comment]
> This is possibly related to a seperate camera issue that I recently discovered where any instance of the camera feed being shown will result in rapid flickering

Indeed. The same happens on Omega Ruby when you try to play/pet pokémon. Camera and microphone notification dot shows up on notification bar and start flickering like crazy, like if Azahar is trying to run them, crashing, and retrying, nonstop. This ends up killing up the process.
